src/Clustering.jl

```diff
@@ -54,7 +54,11 @@ module Clustering
     Hclust, hclust, cutree,
 
     # MCL
-    mcl, MCLResult
+    mcl, MCLResult,
+
+    # V-measure
+    vmeasure
```

Comment:
Although exported names doesn't look strictly ordered, it still would be better to put it after randindex.